pkg/packer/manifests/aws/packer.json.tmpl

```diff
@@ -181,6 +188,7 @@
       "format_options": "{{user `goss_format_options`}}",
       "goss_file": "{{user `goss_entry_file`}}",
       "inspect": "{{user `goss_inspect_mode`}}",
+      "skip_install": false,
```

Comment:
should we need to default it with `true` or read it from parameter?

Reply:
i meant to write true :doh: